app/models/chess_piece.rb

```diff
@@ -10,6 +10,16 @@ def vertical_move?(endpoint_x, endpoint_y)
     (position_y != endpoint_y) && (position_x === endpoint_x)
   end
 
+  def vertically_obstructed?(endpiont_x, endpoint_y)
```

Comment:
you have a typo here.

I found it by looking at the error the specs gave us.
![image](https://cloud.githubusercontent.com/assets/5273755/13380904/ad74b394-de02-11e5-8f61-e1469e44a8b9.png)